shell/platform/darwin/ios/framework/Source/FlutterAppDelegateTest.mm

```diff
@@ -0,0 +1,41 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterAppDelegate.h"
+#import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterEngine.h"
+#import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterAppDelegate_Test.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Test.h"
+
+FLUTTER_ASSERT_ARC
+
+@interface FlutterAppDelegateTest : XCTestCase
+@end
+
+@implementation FlutterAppDelegateTest
+
+- (void)testLaunchUrl {
```

Comment:
Just curious, where do we mock FlutterDeepLinkingEnabled in this test?

Reply:
I didn't, that's why it was failing.  Fixed it.